massgov/mayflower
# Patternlab/DP-12998  remove feedback from print

## Description
Remove the "did you find what you're looking for" component at the bottom of the page from print styles for all pages.
Removed feedback form and button from mobile

## Related Issue / Ticket

- [DP-12998](https://jira.mass.gov/browse/DP-12998)

## Steps to Test


1. Visit any page in Drupal and on printing, see that the feedback form is no longer present.
2. In a mobile view (< 620px) see that the feedback form and button are not present.
